src/OrbitGl/GraphTrack.cpp

```diff
@@ -38,7 +39,8 @@ float GraphTrack<Dimension>::GetHeight() const {
   float height = layout_->GetTrackTabHeight() + GetLegendHeight() +
                  layout_->GetTextBoxHeight() * scale_factor +
                  layout_->GetSpaceBetweenTracksAndThread() +
-                 layout_->GetEventTrackHeightFromTid(thread_id_) + layout_->GetTrackBottomMargin();
+                 layout_->GetEventTrackHeightFromTid(orbit_base::kInvalidThreadTid) +
+                 layout_->GetTrackBottomMargin();
```

Comment:
I assume this works because a GraphTrack has no thread_assigned anyway and so `thread_id_` is effectively always equal to `kInvalidThreadTid`, right?

Reply:
There is no thread_id_ anymore in Track. 

Any constant works there (except the AllThreadsEventTrack) because the TID there is only used for making the height bigger in that case. But I'm changing there and putting a default value inside the method in Layout and I will add Tori as a reviewer, who is the one who created that method.

Reply:
```Any constant works there (except the AllThreadsEventTrack) because the TID there is only used for making the height bigger in that case.```

That's what I meant!. Thank you! LGTM!

Reply:
Adding @vwbaker . At first I used the constant in GraphTrack and now I set a default parameter in Layout for GetEventTrackHeightFromTid(). What do you think it is better?

Reply:
LGTM :)